src/scripts.js

```diff
@@ -8,86 +8,100 @@ const searchPage = document.querySelector('.search-page')
-  searchPage.classList.remove('hidden')
-  homePage.classList.add('hidden')
+  loadPage(searchPage, homePage);
   array.map(recipe => searchPage.innerHTML += `
-    <article class="recipe-card" id="${recipe.id}">
+    <article class="recipe-card recipe ${createKebab(recipe.name)} ">
       <img class="recipe-card-img" src="${recipe.image}">
       <p class="recipe-card-name">${recipe.name}</p>
       <p class="recipe-card-cost">${recipe.getIngredientsCost()}</p>
       <button class="recipe-card-button"></button>
     </article>
-  `)
-}
+  `);
+};
 
-function compileRecipeRepository() {
-  recipeRepository = new RecipeRepository(recipeData, ingredientsData)
-}
-
-function populateRecipeCarousel() {
+const populateRecipeCarousel = () => {
   for (i = 0; i < 5; i++) {
-    let randomRecipe = recipeRepository.recipes[Math.floor(Math.random() * recipeRepository.recipes.length)]
+    const randomRecipe = recipeRepository.recipes[Math.floor(Math.random() * recipeRepository.recipes.length)];
     recipeCarousel.innerHTML += `
-      <article class="recipe-card" id="${randomRecipe.id}">
+      <article class="recipe-card recipe ${createKebab(randomRecipe.name)}" >
         <img class="recipe-card-img" src="${randomRecipe.image}">
         <p class="recipe-card-name">${randomRecipe.name}</p>
         <p class="recipe-card-cost">${randomRecipe.getIngredientsCost()}</p>
         <button class="recipe-card-button"></button>
       </article>
     `
-  }
-}
+  };
+};
 
-function searchAllRecipes(event) {
+const searchAllRecipes = (event) => {
   if (event.key === "Enter" && searchBox.value) {
-    event.preventDefault()
+    event.preventDefault();
     loadSearchPage(recipeRepository.masterSearch(searchBox.value));
-  }
-}
\ No newline at end of file
+  };
+};
+
+const suggestRecipes = () => {
+  document.querySelector('.meal-suggestion-container').innerHTML = "";
+  for (i = 0; i < 3; i++) {
+    const randomRecipe = recipeRepository.recipes[Math.floor(Math.random() * recipeRepository.recipes.length)];
+    document.querySelector('.meal-suggestion-container').innerHTML += `
+      <article class="meal-suggestion recipe ${createKebab(randomRecipe.name)}">
+        <div class="img-cropper">
+          <img class="zoom meal-suggestion-img" src="${randomRecipe.image}">
+        </div>
+        <p class="meal-suggestion-name">${randomRecipe.name}</p>
+      </article>
+    `
+  };
+};
+
+window.addEventListener('load', compileRecipeRepository);
+window.addEventListener('load', populateRecipeCarousel);
+recipeCarousel.addEventListener('click', () => loadRecipeCard(event));
+searchPage.addEventListener('click', () => loadRecipeCard(event));
+document.addEventListener('keydown', searchAllRecipes);
+allRecipesButton.addEventListener('click', () => loadSearchPage(recipeRepository.recipes));
+pageTitle.addEventListener('click', () => loadPage(homePage, searchPage));
+mealSuggestionContainer.addEventListener("click", () => loadRecipeCard(event));
```

Comment:
Thanks for tidying this mess up and moving these event listeners down. Much more readable